styx-service-common/src/test/java/com/spotify/styx/util/ReplayEventsTest.java

```diff
@@ -100,7 +100,7 @@ public void restoreRunStateWhenMissingEvent() throws Exception {
     events.add(SequenceEvent.create(
         Event.triggerExecution(WORKFLOW_INSTANCE, Trigger.backfill("bf-1"), TRIGGER_PARAMETERS),      1L, 1L));
     events.add(SequenceEvent.create(Event.dequeue(WORKFLOW_INSTANCE, RESOURCE_IDS),                   2L, 2L));
-    // missing Event.submit(WORKFLOW_INSTANCE, EXECUTION_DESCRIPTION, "exec-1")
+    events.add(SequenceEvent.create(Event.submit(WORKFLOW_INSTANCE, EXECUTION_DESCRIPTION, "exec-1"), 3L, 3L));
```

Comment:
This was done on purpose in https://github.com/spotify/styx/pull/609, as reflected the the name of this test case.